### PR TITLE
Here is an implementation of the fizzBuzz algorithm using Swift 5 ".isMultiple" function with a Switch statement:

### PR DESCRIPTION
**Pull request by IlijaMihajlovic**
### Checklist

- [x] I've looked at the [contribution guidelines](https://github.com/raywenderlich/swift-algorithm-club/blob/master/.github/CONTRIBUTING.md).
- [x] This pull request is complete and ready for review.

### CODE REVIEW

**Comment by IlijaMihajlovic** [comment]
Here is an implementation of the fizzBuzz algorithm using Swift 5 ".isMultiple" function with a Switch statement

**Comment by IlijaMihajlovic** [comment]
Here is an implementation of the fizzBuzz algorithm using Swift 5 ".isMultiple" function with a Switch statement.

File: Fizz Buzz/README.markdown [hunk 1]
```diff
@@ -55,39 +55,53 @@ Finding numbers divisible by five:
...
-  for i in 1...numberOfTurns {
-    var result = ""
-
-    if i % 3 == 0 {
-      result += "Fizz"
-    }
-
-    if i % 5 == 0 {
-      result += (result.isEmpty ? "" : " ") + "Buzz"
+(0...50).forEach { (num) in
```
Comment (kelvinlauKL) [inline Fizz Buzz/README.markdown]: I'm not a big fan of this. The range is now hard coded for 50 iterations, whereas the previous function allows the user to inject an arbitrary number of iterations.

File: Fizz Buzz/README.markdown [hunk 2]
```diff
@@ -55,39 +55,53 @@ Finding numbers divisible by five:
...
+    
+    switch num {
+        //Check if 'x' is multiple of 3 and if 'x' is multiple of 5. If yes print fizzBuzz!
+    case let x where x.isMultiple(of: 3) && x.isMultiple(of: 5):
```
Comment (kelvinlauKL) [inline Fizz Buzz/README.markdown]: `isMultiple(of:)` is a good addition, but I'm not sure about having these pattern matching statements.

`if` and `else` statements are a lot more relatable to people. What motivated you to try this approach?

**Pull request closed without merging**